static/.gitignore

```diff
@@ -1,2 +1,10 @@
 min/
 source-map/
+/js/bundle.js
```

Comment:
Indeed -- looks like 3f5d0e69f did that and didn't do a complete grep for references. I'll remove